Sources/Brave/Frontend/Browser/Playlist/Controllers/PlaylistListViewController+TableViewDelegate.swift

```diff
@@ -250,7 +264,7 @@ extension PlaylistListViewController: UITableViewDelegate {
                 UIAction(
                   title: Strings.PlayList.sharePlaylistOpenInNewTabTitle, image: UIImage(systemName: "plus.square.on.square"),
                   handler: { [weak self] _ in
-                    self?.openInNewTab(currentItem, isPrivate: false)
+                    self?.openInPrivateTabWithAuthentication(currentItem)
```

Comment:
This is the wrong action, you want the one just below this (`Strings.PlayList.sharePlaylistOpenInNewPrivateTabTitle`)

Reply:
Missed the testing that action sorry from table context menu, it is fixed with https://github.com/brave/brave-ios/pull/7959/commits/6450920c972b039742a2dbde1964b8ead3b45f34